frontend/tests/controllers/QualityNominationTest.php

```diff
@@ -1821,13 +1821,13 @@ public function testAggregateFeedback() {
         $tags1 = array_map($extractName, $tagArrayForProblem1);
         $this->assertEquals(
```

Comment:
```suggestion
        $this->assertEqualsCanonicalizing(
```

[`assertEqualsCanonicalizing()`](https://phpunit.readthedocs.io/en/9.0/assertions.html#assertequalscanonicalizing) no requiere que los elementos estén en el mismo orden! igual abajo.